content/en/docs/ops/ambient/install/istioctl-installation/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Install with Istioctl
+description: Install Istio in Ambient mode with Istioctl.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+This guide shows you how to install Istio in ambient mode with Istioctl.
+Aside from following the demo in [Getting Started with Ambient Mode](/docs/ops/ambient/getting-started/),
+
+## Prerequisites
+
+1. Check the [Platform-Specific Prerequisites](/docs/ops/ambient/install/platform-prerequisites).
+
+1. Install the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.
```

Comment:
latest contradicts saying 1.21.0. Also 1.22 is beta.

Reply:
(these are copy/paste errors from the current doc, which also hasn't been updated; working on that now)